test/unit/query/builder.js

```diff
@@ -750,6 +750,32 @@ describe('QueryBuilder', function() {
     );
   });
 
+  it('clear limits and offsets', function() {
```

Comment:
In general it is advisable to have more granular tests; would you consider splitting this into two tests, separately for limit and for offset?

Reply:
Sorry, i have no more time...